triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerDao.scala

```diff
@@ -100,9 +99,9 @@ object TriggerDao {
     delete.update.run.map(_ == 1)
   }
 
-  def getTriggersForParty(partyToken: Jwt): ConnectionIO[Vector[UUID]] = {
+  def getTriggersForParty(partyToken: String): ConnectionIO[Vector[UUID]] = {
     val select = Fragment.const("select trigger_instance from running_triggers")
-    val where = Fragment.const(s" where party_token = '${partyToken.value}'")
+    val where = Fragment.const(s" where party_token = '${partyToken}'")
```

Comment:
This is where the token is used in a query and we need to make sure it's validated. I think we should only use the username here which we could restrict the charset to that of parties.

Reply:
as discussed (and more or less resolved above) let's stick with tokens but yes, let us make sure at the point we receive the token that the party identifier meets syntactic requirements.

Reply:
Yes but also the encrypted token needs to be within a reasonable character set for this query to work.

Reply:
yes. the token stored in the db will be base 64 thus only a-z, A-Z, 0-9, + and / chars.